pkgs/servers/mail/mailman/default.nix

```diff
@@ -1,23 +1,50 @@
-{ lib, buildPythonPackage, fetchPypi, fetchpatch, isPy3k, alembic, aiosmtpd, dnspython
+{ lib, buildPythonPackage, fetchPypi, fetchpatch, isPy3k, python3, postfix, lynx
+, python-editor
+, aiosmtpd, dnspython
 , flufl_bounce, flufl_i18n, flufl_lock, lazr_config, lazr_delegates, passlib
 , requests, zope_configuration, click, falcon, importlib-resources
-, zope_component, lynx, postfix, authheaders, gunicorn
+, zope_component, authheaders, gunicorn
+#, alembic, sqlalchemy
 }:
 
+let
+  # Mailman does not support sqlalchemy >= 1.4 https://gitlab.com/mailman/mailman/-/issues/899
+  pythonOverride = python3.override {
+    packageOverrides = self: super: {
+      sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
+        version = "1.3.24";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "ebbb777cbf9312359b897bf81ba00dae0f5cb69fba2a18265dcc18a6f5ef7519";
+        };
+      });
+      alembic = super.alembic.overridePythonAttrs (oldAttrs: rec {
+        version = "1.6.5";
+        propagatedBuildInputs = oldAttrs.propagatedBuildInputs ++ [ python-editor ];
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "sha256-oh/t67P7j2u7ulGhERTwjHhwk3cFE4TJxerVcF7pOlE=";
+        };
+      });
+    };
+  };
```

Comment:
Please try a similar `substituteInPlace` thing as suggested in my other comment.

Reply:
since making this pr arch has applied a similar patch https://github.com/archlinux/svntogit-community/commit/8cf572ed315b30dbbec0ba821758ecc3db6cb064#diff-37538beb61ff63edebbf735dfcf39e5d732f49183d6beb097169d971875ca422